specs/core/0_beacon-chain.md

```diff
@@ -169,7 +169,7 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
 | `LATEST_RANDAO_MIXES_LENGTH` | `2**13` (= 8,192) | randao mixes |
 | `EMPTY_SIGNATURE` | `[bytes48(0), bytes48(0)]` | - |
 
-* For the safety of crosslinks a minimum committee size of 111 is [recommended](https://vitalik.ca/files/Ithaca201807_Sharding.pdf). (Unbiasable randomness with a Verifiable Delay Function (VDF) will improve committee robustness and lower the safe minimum committee size.) The shuffling algorithm generally ensures (assuming sufficient validators) committee sizes at least `TARGET_COMMITTEE_SIZE // 2`.
+* For the safety of crosslinks a minimum committee size of 111 is [recommended](https://vitalik.ca/files/Ithaca201807_Sharding.pdf). (Unbiasable randomness with a Verifiable Delay Function (VDF) will improve committee robustness and lower the safe minimum committee size.) The shuffling algorithm generally ensures (assuming sufficient validators) committee sizes at least `TARGET_COMMITTEE_SIZE`.
```

Comment:
I found your explanation (https://github.com/ethereum/eth2.0-specs/pull/341#issuecomment-448757497) is easier to understand. Can we adjust this paragraph to be like:


* For the safety of crosslinks, `TARGET_COMMITTEE_SIZE` is the committee size that exceeds [the recommended minimum committee size `111`](https://vitalik.ca/files/Ithaca201807_Sharding.pdf); with sufficient active validators, the shuffling algorithm ensures committee sizes at least `TARGET_COMMITTEE_SIZE`. Unbiasable randomness with a [Verifiable Delay Function (VDF)](https://vdfresearch.org) will improve committee robustness and lower the safe minimum committee size.